example/books/ex1/books1.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"os"
+
+	"books"
+	"capnproto.org/go/capnp/v3"
+)
+
+func main() {
+	// Make a brand new empty message.  A Message allocates Cap'n Proto structs.
+	msg, seg, err := capnp.NewMessage(capnp.SingleSegment(nil))
+	if err != nil {
+		panic(err)
+	}
+
+	// Create a new Book struct.  Every message must have a root struct.
+	book, err := books.NewRootBook(seg)
+	if err != nil {
+		panic(err)
+	}
+	book.SetTitle("War and Peace")
```

Comment:
nit:  String setters return an error.  It's totally okay to ignore it for a demo, but I think we should signal to readers that the error exists, and should be handled in production code.

Suggest:  `_ = book.SetTitle("War and Peace")`

Reply:
Fixed in 48ca3918cc0b56e42c5119d884edbbf98a26f2c3